types/node/index.d.ts

```diff
@@ -3944,8 +3944,17 @@ declare module "util" {
     export function puts(...param: any[]): void;
     export function print(...param: any[]): void;
     export function log(string: string): void;
-    export function inspect(object: any, showHidden?: boolean, depth?: number | null, color?: boolean): string;
-    export function inspect(object: any, options: InspectOptions): string;
+    export var inspect: {
+        (object: any, showHidden?: boolean, depth?: number | null, color?: boolean): string;
+        (object: any, options: InspectOptions): string;
+        colors: {
+            [color: string]: [number, number]
+        }
+        styles: {
+            [style: string]: string
+        }
+        custom: symbol
+    }
```

Comment:
Please consider to add also `defaultOptions` (https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_inspect_defaultoptions).